### DISCUSSION TITLE
Powered by chatgpt-web v2.14.0-b8bc056，这个标改改位置撒。

### DISCUSSION THREAD

**Discussion by xiaosaaaa** [chatgpt-web-dev/chatgpt-web#462] [General]
太影响美观了强迫症受不了啦。

**Comment by BobDu** [comment]
有一个想法是在设置页面。做一个`关于`选项卡。在里面写具体的版本信息等。 
然后在 5xx 页面。也展示 版本信息等。
如果有社区贡献者能提交一个PR帮助实现的话。相信可以更快推进这一变化😊

**Comment by 0verleaves** [comment]
@BobDu 请问为什么我的版本号是v0.0.0-xxxxxxx
![image](https://github.com/chatgpt-web-dev/chatgpt-web/assets/54617500/5ab12d6b-9979-4975-bd6b-7e6f866ea514)
